src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1093,20 +1093,42 @@ namespace UnitarySystems {
                     int ControlNode = this->m_SystemCoolControlNodeNum;
                     if (ControlNode > 0) {
                         this->checkNodeSetPoint(state, AirLoopNum, ControlNode, CoolingCoil, OAUCoilOutTemp);
+                    } else if (this->m_ControlType == UnitarySysCtrlType::Setpoint) {
+                        ShowSevereError(state, format("{}: Missing set point in {} = {}", routineName, this->UnitType, this->Name));
+                        ShowContinueError(state,
+                                          format("...Setpoint is required at system air outlet node = {} or cooling coil air outlet node = {}",
+                                                 state.dataLoopNodes->NodeID(this->AirOutNode),
+                                                 state.dataLoopNodes->NodeID(this->CoolCoilOutletNodeNum)));
+                        state.dataUnitarySystems->initUnitarySystemsErrorsFound = true;
```

Comment:
Suggested location for warnings seems appropriate. Used existing initUnitarySystemsErrorsFound bool to set when warning occurs. This bool is used throughout Init and should not typically be set unless there is an issue with calls using this bool. For example:

    this->m_DesignSuppHeatingCapacity = SteamCoils::GetCoilCapacity(
        state, "Coil:Heating:Steam", this->m_SuppHeatCoilName, state.dataUnitarySystems->initUnitarySystemsErrorsFound);

This call, and others like it, should be clean in Init since coils have been validated in getInput.